ui/components/app/collectible-details/index.scss

```diff
@@ -66,4 +69,9 @@ $spacer-break-small: 16px;
     flex: 0 0 $link-title-width;
     max-width: 0 0 $link-title-width;
   }
+
+  &__send-button {
+    margin-inline-end: 8px;
+    max-width: 250px;
+  }
```

Comment:
One more thing I noticed while going over the designs is that the send button should be `160px` in expanded view and fullwidth in pop up view

Screenshots from Figma designs
<img width="692" alt="Screen Shot 2022-01-08 at 12 17 11 PM" src="https://user-images.githubusercontent.com/8112138/148619383-56f7ab73-a1ce-44f6-a2d5-4a7f8c558935.png">
<img width="525" alt="Screen Shot 2022-01-08 at 12 17 30 PM" src="https://user-images.githubusercontent.com/8112138/148619394-c7953d36-3b14-4535-8646-964f15b1f270.png">


```suggestion
  &__send-button {
    margin-inline-end: 8px;

    @media screen and (min-width: $break-large) {
      max-width: 160px;  
    }
  }
```

Reply:
done here: https://github.com/MetaMask/metamask-extension/pull/13048/commits/2a8884490bc45658b31021268b45a219233ea944

Reply:
I don't see the changes in the commit listed above?

Reply:
Sorry wrong link. Here's the right one: https://github.com/MetaMask/metamask-extension/pull/13048/commits/8771e9b24ad76f82f774e5b26f46cbfe6c243b07

Reply:
confirmed changed